enginetest/queries/queries.go

```diff
@@ -8588,10 +8605,11 @@ var ErrorQueries = []QueryErrorTest{
 		Query:       `SELECT * FROM mytable WHERE s REGEXP("*main.go")`,
 		ExpectedErr: expression.ErrInvalidRegexp,
 	},
-	{
-		Query:       `SELECT SUBSTRING(s, 1, 10) AS sub_s, SUBSTRING(SUB_S, 2, 3) AS sub_sub_s FROM mytable`,
-		ExpectedErr: sql.ErrMisusedAlias,
-	},
+	// todo: we should be case-sensitive and reject SUB_S as a valid field
```

Comment:
Since when are alias names case sensitive?

Reply:
hmm yeah this is wrong, thanks for pointing out, it's fixed now